src/main/drivers/motor.c

```diff
@@ -138,6 +138,14 @@ static void analogInitEndpoints(const motorConfig_t *motorConfig, float outputLi
     }
 }
 
+bool isProtocolBidirectionalDshot(const motorDevConfig_t *motorDevConfig)
+{
+    bool motorProtocolDshot = false;
+    const bool motorProtocolEnabled = checkMotorProtocolEnabled(motorDevConfig, &motorProtocolDshot);
+    const bool isBidir = motorDevConfig->useDshotTelemetry;
+    return motorProtocolEnabled && motorProtocolDshot && isBidir;
```

Comment:
```suggestion
bool isMotorProtocolBidirDshot(const motorDevConfig_t *motorDevConfig)
{
    return isMotorProtocolDshot() && motorDevConfig->useDshotTelemetry;
```
This is sufficient (because if `isMotorProtocolDshot()` is true, `isMotorProtocolEnabled()` will also be true). And can you move this function to the others (to line 305)?